src/entity/Entity.php

```diff
@@ -1535,6 +1557,19 @@ public function despawnFromAll() : void{
 		}
 	}
 
+	/**
+	 * Returns the item that players will equip when middle-clicking on this entity.
+	 * If addUserData is true, additional data may be added, such as name tag, fire ticks, etc.
+	 */
+	public function getPickedItem(bool $addUserData) : Item{
+		$item = ItemFactory::getInstance()->get(ItemIds::SPAWN_EGG, LegacyEntityIdToStringIdMap::getInstance()->stringToLegacy(static::getNetworkTypeId()));
```

Comment:
I can't accept this.

Reply:
What could I replace this for? I didn't find anything else that could convert entity identifiers to spawn egg meta values.

Reply:
You shouldn't be using either of those things. The subclasses should declare which spawn egg type should be used.